client/scss/layouts/_groups-edit.scss

```diff
@@ -24,9 +24,3 @@
     padding: 0;
   }
 }
-
-.page-permissions-listing {
-  .admin_page_chooser .field-content {
-    width: 100%; // so that 'choose another page' button displays in its entirety
-  }
-}
```

Comment:
As far as I can tell this is no longer needed with the short button labels (same for workflows).